winui/Rating/Precision.md

```diff
@@ -10,3 +10,86 @@ documentation: ug
 # Precision in WinUI Rating
 
 This section explains the precision features available in the WinUI Rating control.
+
+## Rating Control using the Precision property
+
+The Rating control Precision can be customized using the `Precision` Property. It have three types Full, Half and Exact. The default value is "Full".  
```

Comment:
@Priyanka-Vijayasankar The rating value can be provided with three options such as full, half, and exact using the `Precision` property. The default value is `Full`.